test/integration/generated/body-file/Source/AutoRestSwaggerBatFileClient.swift

```diff
@@ -33,6 +33,25 @@ public final class AutoRestSwaggerBatFileClient: PipelineClient {
 
     // MARK: Initializers
 
+    /// Create a AutoRestSwaggerBatFileClient client.
+    /// - Parameters:
+    ///   - authPolicy: An `Authenticating` policy to use for authenticating client requests.
+    ///   - options: Options used to configure the client.
+    public convenience init(
+        authPolicy: Authenticating,
+        withOptions options: AutoRestSwaggerBatFileClientOptions
+    ) throws {
+        // let components = URLComponents()
+        // components.scheme = "http"
+        // components.host = self.host
+        // self.init(baseUrl: components.url, authPolicy: authPolicy, withOptions: options)
```

Comment:
Why is this commented out? This was generated commented out? Looking at `ClientConvenienceInitSnippet` I don't see where this is coming from.

Reply:
Regenerated the code and push to new iteration.